capstone-frontend/src/components/BookDescriptionOverlay.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { Popover, OverlayTrigger } from 'react-bootstrap';
+
+const popoutDescription = (book) => {
```

Comment:
This looks like it should really be treated as a stateless functional component, since it returns JSX.

Reply:
Treating this as a stateless functional component caused the popout to render at the top left of the screen instead of next to the book the user was clicking on. 

Discussed this offline and was not able to figure out a way to use the PopoutDescription component in the overlay call and produce the desired result. Left a comment above the function explaining the reasoning.